build/commands/lib/gitPatcher.js

```diff
@@ -160,9 +171,14 @@ module.exports = class GitPatcher {
     // Detect if any of the files the patch applies to have changed
     for (const {path: localPath, checksum} of appliesTo) {
       const fullPath = path.join(this.repoPath, localPath)
-      const currentChecksum = await calculateFileChecksum(fullPath)
-      if (currentChecksum !== checksum) {
-        return patchApplyReasons.SRC_CHANGED
+      try {
+        await fs.access(fullPath)
+        const currentChecksum = await calculateFileChecksum(fullPath)
+        if (currentChecksum !== checksum) {
+          return patchApplyReasons.SRC_CHANGED
+        }
+      } catch (err) {
```

Comment:
this will incorrectly treat `calculateFileChecksum` exceptions as "file removed". Not sure if we really want it this way.

Reply:
Fixed. Thanks.